src/arch/arm64/vm.c

```diff
@@ -0,0 +1,490 @@
+#if !defined(__aarch64__) || !defined(__linux__)
+#error "This implementation is dedicated to Linux/aarch64."
+#endif
+
+#include <asm/kvm.h>
+#include <assert.h>
+#include <fcntl.h>
+#include <libfdt.h>
+#include <linux/kvm.h>
+#include <linux/kvm_para.h>
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <string.h>
+#include <sys/ioctl.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <unistd.h>
+
+#include "err.h"
+#include "vm-arch.h"
+#include "vm.h"
+
+struct vm_arch_priv {
```

Comment:
Make use of typedef. i.e.,
```c
typedef struct {
    uint64_t entry;
    size_t initrdsz;
    int gic_fd;

    /* This device is a bridge between mmio_bus and io_bus*/
    struct dev iodev;
} vm_arch_priv_t;
```

Then, replace all occurrence `strut vm_arch_priv` with `vm_arch_priv_t`.